Modules/vc-module-content/VirtoCommerce.ContentModule.Web/ExportImport/BackupObject.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections.Generic;
+using VirtoCommerce.ContentModule.Core.Model;
+
+namespace VirtoCommerce.ContentModule.Web.ExportImport
```

Comment:
Remove this class, switch to streamed export/import
See as Export/Import done  for other modules